control-plane/connect-inject/webhook/container_volume.go

```diff
@@ -10,11 +10,15 @@ const volumeName = "consul-connect-inject-data"
 
 // containerVolume returns the volume data to add to the pod. This volume
 // is used for shared data between containers.
-func (w *MeshWebhook) containerVolume() corev1.Volume {
+func (w *MeshWebhook) containerVolume(isWindowsPod bool) corev1.Volume {
+	storageMedium := corev1.StorageMediumMemory
+	if isWindowsPod {
```

Comment:
Isn't the problem with GKE + Windows combination? Are we good with having the same storage medium in AWS and Azure?

Reply:
yes we discussed this with @ishustava. we will do this for all windows